src/pages/volunteer page/components/CheckboxesTree/styles.css

```diff
@@ -0,0 +1,29 @@
+
+.list-container {
+  /* width: 30vw;
+  height: 20vh; */
+}
+
+ul {
```

Comment:
please check with meir if the default tags style in same file (styles.css) i think he did it in base.css or something